pandas/core/indexes/datetimes.py

```diff
@@ -1968,14 +1972,36 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
             .. deprecated:: 0.15.0
                Attempt to infer fall dst-transition hours based on order
 
+
         Returns
         -------
         localized : DatetimeIndex
 
+        Examples
+        --------
+        In the example below, We put the date range from 01 March 2018
+        to 08 March 2018 & localize this to US/Eastern Time zone & again
+        we perform reverse operation where we remove tize zone & make it
+        tz-naive
+
+        >>> dti = pd.date_range('2018-03-01', '2018-03-05')
+        >>> dti
+        DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03',
+        '2018-03-04', '2018-03-05'], dtype='datetime64[ns]', freq='D')
+        >>> dti.tz_localize(tz='US/Eastern')
+        DatetimeIndex(['2018-03-01 00:00:00-05:00',
+        '2018-03-02 00:00:00-05:00', '2018-03-03 00:00:00-05:00',
+        '2018-03-04 00:00:00-05:00', '2018-03-05 00:00:00-05:00'],
+        dtype='datetime64[ns, US/Eastern]', freq='D')
+        >>> dti.tz_localize(None)
```

Comment:
this last won't work, you need to do
``dti.tz_localize('US/Eastern').tz_localize(None)``, which gets you back the original

Reply:
![screenshot from 2018-03-10 21-00-32](https://user-images.githubusercontent.com/22932421/37243850-22374e6c-24a6-11e8-9fdb-aca1b1c4593f.png)
it works well

Reply:
it works but its not very useful. The point is to take a tz-aware, IOW some you previously used ``tz_localize`` on, or just constructed directly and the remove the tz. 

How about you construct a new dti with a tz (IOW pass directly to ``date_range``), then use ``.tz_localize(None)`` on it.

Reply:
@jreback both gives same result

Reply:
oh K Will do that